integration/fuse/bin/alluxio-fuse

```diff
@@ -63,6 +63,7 @@ mount_fuse() {
     mount_point="$1"
     mount_alluxio_path="$2"
   elif [[ $# -eq 1 ]]; then
+    echo "Only argument mount point is found. Alluxio path will be defined by the value of property alluxio.fuse.mount.alluxio.path."
```

Comment:
If there's only one argument given, let the user know we are using it as the mount point.

Reply:
be defined by -> be set to? or will use?